ParabolInc/parabol
# chore: update sign up redirect destination

Fixes #8941

**How to test:**

- Sign up with password
- See that in 50% cases you redirected to the /team/{ID} page instead of /meetings
- See that login to existing account always redirects to /meetings
- Test the same with sing up with google
